src/content/developers/docs/consensus-mechanisms/pos/keys/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Keys in proof-of-stake Ethereum
+description: An explanation of keys used in Ethereum since the transition to proof-of-stake.
```

Comment:
I think we can explain (most) things as how they are now (post-merge) vs relative to how they were.

```suggestion
description: An overview of keys used in Ethereum.
```

Reply:
Similar to above, I think keeping it dialed into PoS is appropriately here.
```suggestion
description: An explanation of keys used in Ethereum's proof-of-stake consensus mechanism
```